app/views/principals/pre_qualification_form.html.erb

```diff
@@ -1,119 +1,129 @@
-<div class="l-constrained">
```

Comment:
This is a terrible diff. Essentially:

* The original content has been **indented** and placed into an `l-2col-main`
* A new `<section>` has been added before it to render the `sign_in_panel` partial
* A new `l-1col` section has been added to place the introductory elements evenly above the columns below.

Any questions, just ask.